schema-dex-amm.graphql

```diff
@@ -81,13 +82,14 @@ interface Protocol {
 
   type: ProtocolType!
 
-  " # of total/cumulative unique users "
-  totalUniqueUsers: Int!
+  " Current TVL (Total Value Locked) of the entire protocol "
+  currentTvlUSD: BigDecimal!
 
-  totalValueLockedUSD: BigDecimal!
+  " # of cumulative unique users "
+  cumulativeUniqueUsers: Int!
 
-  " Total volume in USD "
-  totalVolumeUSD: BigDecimal!
+  " All historical volume in USD "
+  cumulativeVolumeUSD: BigDecimal!
```

Comment:
I have a doubt regarding this one: This is all historical traded volume, right?

Reply:
none, you just answered my doubt :) thanks (I just wanted to make sure I understood this)